packages/sakuli-cli/src/commands/enable-enterprise/tasks/license-tasks.function.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { licenseGlobalTask } from "./license-tasks.function";
+import execa = require("execa");
+import { promises as fs } from 'fs';
+import { sep, join } from "path";
+import { homedir } from "os";
+import rimraf = require("rimraf");
+jest.mock('execa', () => jest.fn());
+jest.mock('os', () => ({
+    homedir: jest.fn()
+}))
+const { tmpdir } = jest.requireActual('os');
+
+describe('licenseGlobalTask', () => {
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    })
+
+    describe('on win', () => {
+
+        let platform: string;
+        beforeEach(() => {
+            platform = process.platform;
+            Object.defineProperty(process, 'platform', {
+                value: 'win32'
+            })
+        })
+
+        it('should use setX command to set env var', async () => {
+            // GIVEN
+            const task = licenseGlobalTask('abcdefg');
+
+            // WHEN
+            await task();
+
+            // THEN
+            expect(execa).toHaveBeenCalledWith('setx', ['SAKULI_LICENSE_KEY=abcdefg'])
+
+        })
+
+        afterEach(() => {
+            Object.defineProperty(process, 'platform', {
+                value: platform
+            })
+        })
+
+    })
+    describe('on unix', () => {
+
+        let platform: string;
+        let tmpHomeDirMock: string;
+        beforeEach(async () => {
+            platform = process.platform;
+            Object.defineProperty(process, 'platform', {
+                value: 'linux'
+            })
+            tmpHomeDirMock = await fs.mkdtemp(`${tmpdir()}${sep}`);
+            (<jest.Mock>homedir).mockReturnValue(tmpHomeDirMock);
+        })
+
+        it('should use add license env var to .bashrc', async () => {
```

Comment:
We have to keep in mind, that this does only work for bash. fish and zsh have different config files.
Maybe this is something we should consider adding here?

```
if(exists ~/.bashrc)
  addSakuliLicense(~/.bashrc)
else if (exists ~/.config/fish/config.fish)
  addSakuliLicense(~/.config/fish/config.fish)
else ...
```

Reply:
Since we have only mentioned `.bashrc`in our docs page I used this approach.

I personally configure everything in `.bashrc`and use `source ~/.bashrc` in all the other configurations... thought that this is a common way?!

When we support fish, zsh how far should we go? I think there a lot of tools with it's own config file right? What about users that installed fish, zsh but use only one (or switch them depending on a use-case).

Reply:
I agree, then let's wait until someone asks for zsh or fish support. :+1: